demisto_sdk/commands/common/tests/lists_test.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+import pytest
+from mock import patch
+
+from demisto_sdk.commands.common.constants import LISTS_DIR, PACKS_DIR
+from demisto_sdk.commands.common.hook_validations.lists import ListsValidator
+from demisto_sdk.commands.common.hook_validations.structure import \
+    StructureValidator
+from demisto_sdk.commands.common.tools import src_root
+
+TEST_DATA = src_root() / 'tests' / 'test_files'
+TEST_CONTENT_REPO = TEST_DATA / 'content_slim'
+LIST_GOOD = TEST_CONTENT_REPO / PACKS_DIR / 'Sample01' / LISTS_DIR / 'list-checked_integrations.json'
+
+
+def mock_structure(file_path=None, current_file=None, old_file=None):
+    with patch.object(StructureValidator, '__init__', lambda a, b: None):
+        structure = StructureValidator(file_path)
+        structure.is_valid = True
+        structure.scheme_name = 'list'
+        structure.file_path = file_path
+        file = open(file_path, "r")
+        structure.current_file = json.loads(file.read())
+        file.close()
+        structure.old_file = old_file
+        structure.prev_ver = 'master'
+        structure.branch_name = ''
+        structure.quite_bc = False
+        return structure
+
+
+class TestListValidator:
+
+    @pytest.mark.parametrize('list_path, is_valid', [(LIST_GOOD, True)])
+    def test_is_valid_list(self, list_path, is_valid):
```

Comment:
add tests for all cases, (fromVersion, fromServerVersion, toVersion, toServerVersion), and invalid cases as well

Reply:
Done